email-question-draft.md

```diff
@@ -0,0 +1,36 @@
+---
+from: Jon (or generic e-mail)
+to: .NET Foundation Members
+subject: .NET Foundation Board Elections - Your Turn to Ask the Questions!
+---
+
+> "Asking questions is the first way to begin change." - Kubra Sait
+
+Foundation Members,
```

Comment:
ℹ️  We could also include "and potential future members" depending on the audience. But I think we might only be able to send e-mails to those who are currently accepted and not in the backlog.